theano/gpuarray/dnn.py

```diff
@@ -1282,6 +1282,11 @@ def dnn_pool(img, ws, stride=(1, 1), mode='max', pad=(0, 0)):
 
     """
     img = gpu_contiguous(img)
+    if mode == "sum":
+        ret = GpuDnnPool(mode="average_inc_pad")(img, ws, stride, pad)
+        context_name = ret.type.context_name
+        window_elem = numpy.prod(ws).astype(ret.dtype)
```

Comment:
ws can be symbolic now, right? It may be the case when the test fails on Jenkins.